realm/config/findbugs/findbugs-filter.xml

```diff
@@ -21,5 +21,10 @@
     <Match>
         <Class name="io.realm.PermissionOfferResponseRealmProxy"/>
     </Match>
+    <Match>
```

Comment:
can we just remove the local var `final Future<?> future = ` at `SyncUser.java:278`? maintaining a long list of findbugs filter is quite annoy.